custom/icds_reports/data_pull/data_pulls.py

```diff
@@ -193,3 +201,77 @@ def _dump_consolidated_data(result):
             row['State'] = state_name
             writer.writerow(row)
         return result_file
+
+
+class VHSNDMonthlyReport(MonthBasedDataPull):
+    slug = "vhsnd_monthly_report"
+    name = "VHSND monthly report"
+    queries = [
+        VHSNDMonthlyCount
+    ]
+    file_format_required = 'xlsx'
+
+    def post_run(self, data_files):
+        state_results = self._consolidate_data(data_files)
+        return self._dump_consolidated_data(state_results)
+
+    def _consolidate_data(self, data_files):
+        result = defaultdict(dict)
+        state_name_column = 'state_name'
+        vhnsnd_date_column = 'vhsnd_date_past_month'
+        for filename, filestream in data_files.items():
+            filestream.seek(0)
+            reader = csv.DictReader(filestream)
+            for row in reader:
+                state_name = row[state_name_column]
+                vhsnd_date = parse_date(row[vhnsnd_date_column])
+                vhsnd_date = vhsnd_date.strftime('%d/%m/%Y')
```

Comment:
nit: i think this can be appended to line above itself.

Reply:
done